src/parsing/lpParser.mly

```diff
@@ -167,9 +168,9 @@ command:
         {p_sym_mod=ms; p_sym_nam=s; p_sym_arg=al; p_sym_typ=ao;
          p_sym_trm=fst tp; p_sym_prf=snd tp; p_sym_def=true}
       in make_pos $sloc (P_symbol(sym)) }
-  | ms=modifier* xs=param_list* INDUCTIVE
+  | xs=param_list* INDUCTIVE
     is=separated_nonempty_list(WITH, inductive) SEMICOLON
-      { make_pos $sloc (P_inductive(ms,xs,is)) }
+      { make_pos $sloc (P_inductive([],xs,is)) }
```

Comment:
I propose to remove the first argument of P_inductive everywhere.

Reply:
Sorry, I made a little mistake. After reviewing the code and documentation, `inductive` type could not use _property_ modifier but they can use _exposition_ modifiers (`private`, `protected`). So I created a subrule `exposition` in the grammar and used it in `modifier` rule and `inductive` case in `command` rule.
I keep a modifier List for `P_inductive` to avoid more modification in this PR, but maybe we can refactor the code in another PR. Moreover, I was not sure that the `constant` modifier was added later in the code to the `ms`, which further encourages keeping a modifier list.